process/misc_windows.go

```diff
@@ -0,0 +1,80 @@
+package process
```

Comment:
does this need a windows build directive or do we only import this module in windows-specific code?

Reply:
That's right - the name of the file tells the compiler to only include it in windows builds. See [go/build package docs](https://golang.org/pkg/go/build/#hdr-Build_Constraints).

Reply:
neat!  I didn't know that